releases/22.0.0.8/full/helpers/runtime/configure-liberty.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# If the Liberty server name is not defaultServer and defaultServer still exists migrate the contents
+if [ "$SERVER_NAME" != "defaultServer" ] && [ -d "/opt/ol/wlp/usr/servers/defaultServer" ]; then
+  # Create new Liberty server
+  /opt/ol/wlp/bin/server create --template=javaee8 >/tmp/serverOutput 
+  rc=$?
+  if [ $rc -ne 0 ]; then
+    cat /tmp/serverOutput
+    rm /tmp/serverOutput
+    exit $rc
+  fi
+  rm /tmp/serverOutput
+
+  # Verify server creation
+  if [ ! -d "/opt/ol/wlp/usr/servers/$SERVER_NAME" ]; then
+    echo "The server name contains a character that is not valid."
+    exit 1
+  fi
+  chmod -R g+w /opt/ol/wlp/usr/servers/$SERVER_NAME
+
+  # Delete old symlinks
+  rm /opt/ol/links/output
+  rm /opt/ol/links/config
+
+  # Add new output folder symlink and resolve group write permissions
+  mkdir -p $WLP_OUTPUT_DIR/$SERVER_NAME
+  ln -s $WLP_OUTPUT_DIR/$SERVER_NAME /opt/ol/links/output
+  chmod g+w $WLP_OUTPUT_DIR/$SERVER_NAME
+  mkdir -p $WLP_OUTPUT_DIR/$SERVER_NAME/resources
+  mkdir -p $WLP_OUTPUT_DIR/$SERVER_NAME/workarea
+  mkdir -p $WLP_OUTPUT_DIR/$SERVER_NAME/logs
+  chmod -R g+w $WLP_OUTPUT_DIR/$SERVER_NAME/workarea
+  chmod -R g+w,o-rwx $WLP_OUTPUT_DIR/$SERVER_NAME/resources
+  chmod -R g+w,o-rwx $WLP_OUTPUT_DIR/$SERVER_NAME/logs
+  
+  # Hand over the SCC
+  if [ "$OPENJ9_SCC" = "true" ] && [ -d "/opt/ol/wlp/output/defaultServer/.classCache" ]; then
+    mv /opt/ol/wlp/output/defaultServer/.classCache $WLP_OUTPUT_DIR/$SERVER_NAME/
+  fi
+  rm -rf /opt/ol/wlp/output/defaultServer
+
+  # Add new server symlink and populate folder
+  mv /opt/ol/wlp/usr/servers/defaultServer/* /opt/ol/wlp/usr/servers/$SERVER_NAME/
```

Comment:
**Note**: Wondering if there is a better way to achieve the above line and the safety of it to end customers.. However, I was able to get petclinic working with the above mv of the /config directory. Also unsure about whether we need to also move contents from /output which also loses its symlink in this process.